CMakeLists.txt

```diff
@@ -107,6 +107,9 @@ foreach(depspec ${PLATFORM_REQUIRED_DEPS})
   # optional subdirectory for the specified platform to get built.
   split_dependency_specification(${depspec} dep version)
   string(TOUPPER ${dep} depup)
+  if(NOT ${ENABLE_${depup}} STREQUAL AUTO AND NOT ${ENABLE_${depup}})
+    message(FATAL_ERROR "You cannot disable the dependency ${dep} required on platform ${CORE_PLATFORM_NAME}. Please choose another platform or add \"-DENABLE_${depup}=ON\" to your CMake command line to fix this problem.")
```

Comment:
Have you already tried with message type `WARNING`?
This gives a multiline output and might not be overlooked that easily.

Reply:
if we error out there the line below that enables the dep is obsolete.
I prefer a warning here instead.

Reply:
because it changes behavior and we don't want that in beta stage.
I assume it would break some build scripts/packaging where the OS/distro sets (sometimes arbitrary) options without the user directly noticing. If the dep is not available, it will fail anyway.

Reply:
> because it changes behavior and we don't want that in beta stage.

Good point. Updated.